contrib/gate/Dockerfile

```diff
@@ -1,19 +1,16 @@
-FROM fedora:29
+FROM fedora:30
```

Comment:
oh good catch.  Yes it's a PITA all these details must be updated separately, some of it is historical, others are needed to support non-CI workflows.

If you have a mind for it, I _think_ the base-image tag can be made to use a defaulting build-variable.  Something like this:

```
ARG  BASE_TAG=29
FROM fedora:$BASE_TAG
...
```

The in automation scripts / Makefiles, run the build with `podman build ... --build-arg BASE_TAG=$SOME_VARIABLE` where `$SOME_VARIABLE` is defined in one place.  Just an idea, not sure if it would work or not.  I seem to remember there being some silly restriction on using those in the `FROM` line, but I may be wrong.

Reply:
okay, I got this staged for my next `--force` push, and I needed to rebase anyway, so re-testing was inevitable.  Thanks for pointing these things out.